sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -988,6 +990,46 @@ def custom_metric(x1, x2):
     assert_array_almost_equal(dist1, dist2)
 
 
+def test_unsupported_metric_for_auto():
```

Comment:
I don't know what you mean by unsupported. It's clearly not unsupported if you're testing it works.